lib/livebook_web/live/home_live.ex

```diff
@@ -31,6 +31,12 @@ defmodule LivebookWeb.HomeLive do
                   <%= remix_icon("compass-line") %>
                 </button>
               </span>
+              <span class="tooltip top" aria-label="Import notebook">
+                <%= live_patch to: Routes.home_path(@socket, :import, "url"),
+                      class: "button button-outlined-gray button-square-icon" do %>
+                  <%= remix_icon("file-upload-line") %>
```

Comment:
I would like to add a text too. Maybe “Import”? Do we need an icon if we have text? 🤔

Reply:
I think it's better to have one prominent button and then icons, here:

![image](https://user-images.githubusercontent.com/17034772/115749820-ca0a6480-a397-11eb-9673-d614b4321772.png)

![image](https://user-images.githubusercontent.com/17034772/115749900-dbec0780-a397-11eb-8dd4-97605cb9d6fb.png)

Let me know which one you prefer.